api/TimeAddressableMediaStore.yaml

```
@@ -1513,13 +1513,13 @@ paths:
           description: |
             A comma separated list of labels of flow segment `get_urls` to include in the response.
             Omitting `accept_get_urls` will result in all `get_urls` returned.
-            A 'default' label selects URLs with a 'default' or no label.
             An empty `accept_get_urls` results in an empty or no `get_urls` in the response.
+            Flow segment `get_urls` with no label cannot be filtered for.
```

Nit: Removes an ambiguity which could be interpreted as "always returned"
```suggestion
            Flow segment `get_urls` with no label cannot be filtered for; they will only be returned if `accept_get_urls` is omitted.
```